src/coreclr/jit/valuenum.cpp

```diff
@@ -291,6 +291,16 @@ VNFunc GetVNFuncForNode(GenTree* node)
                 assert(varTypeIsFloating(node->gtGetOp2()));
                 assert((node->gtFlags & GTF_RELOP_NAN_UN) == 0);
             }
+            else if (node->gtGetOp1()->IsIntegralConst(0))
+            {
+                // Normalize "0 == X" to "0 u>= X"
```

Comment:
```suggestion
                // Normalize "0 == X" to "0 >= X"
```

Is the `u` a typo or does it signify an unsigned comparison?

Reply:
yes, it means "unsigned", I've just added it to other comments